<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-C9T9OGT" failures="1" tests="1" name="ValidateTest" time="27.356" errors="0" timestamp="19 May 2021 10:16:33 GMT">
  <testcase classname="Academy.validateTitleTest" name="basePageNavigation" time="0.083">
    <failure type="java.lang.AssertionError" message="expected [Sign up] but found [Sign in]">
      <![CDATA[java.lang.AssertionError: expected [Sign up] but found [Sign in]
	at Academy.validateTitleTest.basePageNavigation(validateTitleTest.java:29)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- basePageNavigation -->
</testsuite> <!-- ValidateTest -->
